Makefile

```diff
@@ -12,6 +12,9 @@ lint:
 test:
 	@nosetests --exclude="tests_integration" --with-coverage --cover-erase --cover-package=barterdude
 
+test-container:
+	@docker-compose run --rm barterdude nosetests --exclude="tests_integration" --with-coverage --cover-erase --cover-package=barterdude
```

Comment:
It's a good practice, but should include integration tests too. Try with Python 3.7

Reply:
Ok.

Reply:
Done.